src/lib/routes/admin-api/context.ts

```diff
@@ -198,7 +218,10 @@ export class ContextController extends Controller {
         res.status(200).end();
     }
 
-    async validate(req: Request, res: Response): Promise<void> {
+    async validate(
+        req: Request<void, void, { name: string }>,
```

Comment:
We should probably add a request schema to this route as well (like `nameSchema`), so that it shows correctly in the spec. Alternatively we could remove the OpenAPI middleware from this route if it shouldn't show up.